src/workerd/api/node/process.h

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+#pragma once
+
+#include "external/generated_versions/versions.h"
+
+#include <workerd/api/node/i18n.h>
+#include <workerd/jsg/jsg.h>
+#include <workerd/util/mimetype.h>
+
+namespace workerd::api::node {
+
+class ProcessModule final: public jsg::Object {
+ public:
+  ProcessModule() = default;
+  ProcessModule(jsg::Lock&, const jsg::Url&) {}
+
+#ifdef _WIN32
```

Comment:
Even when workerd is built on win32 and mac, I don't think we actually want to expose that fact. Everything should be expected to act generally the same as is does in production. For instance, in our implementation of `node:path` we always use the posix impl regardless.

Reply:
This was actually already there - it's only in the diff since I separated out these process features from util.

It did surprise me as well to see this, it seems it only applies for local workerd workflows. In theory it might be a breaking change for workerd users though?

Reply:
I've gone ahead and removed this to always be `linux`, but that is actually a change from what we had previously which is to provide the platform name - https://github.com/cloudflare/workerd/blob/24d5823cfb16684d0e2a46d31a88a456376cd1b0/src/workerd/api/node/util.h#L165.